src/apurva/Program_1.java

```diff
@@ -0,0 +1,33 @@
+package apurva;
+/*Max repeating word with its frequency.
+input: Hi Hello Hi Techno Hello Hi
+output: Hi -> 3*/
+
+public class Program_1 {
+	void displayCount(String str) {
+		int max = 0, maxWord = 0;
+		String[] strArray = str.split(" ");
+		for (int i = 0; i < strArray.length; i++) {
+			String word = strArray[i];
+			int count = 0;
+			if (str.contains(word)) {
+				for (int j = 0; j < strArray.length; j++) {
```

Comment:
Here you are putting one word from array to the variable called "word", & then you are checking that variable value into array, so here think logically, it will always gives you true right?
storing in one variable and checking same variable in that array makes no sense here.